lib/declare_schema/model/column.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+module DeclareSchema
+  class UnknownSqlTypeError < RuntimeError; end
+
+  module Model
+    # This class is a wrapper for the ActiveRecord::...::Column class
```

Comment:
This class is new. There was too much code in `FieldSpec` that was juggling concerns around `ActiveRecord::...Column`.